ui/app/templates/components/clients/current.hbs

```diff
@@ -54,9 +54,15 @@
         </ToolbarFilters>
       </Toolbar>
     </div>
+    {{! TODO CMB this warning should only show if an upgrade to 1.9 or 1.10 happened in current month }}
```

Comment:
it looks like this is handled? do you still need the todo?

Reply:
The warning is outdated, and needs to be updated (Ivana has updated designs) so that different warnings show based on a 1.9 vs. 1.10 upgrade